cse/reader/CommentReader.py

```diff
@@ -77,14 +81,17 @@ def __parseIterRow(self, row, skipArticleMapping=True):
         commentId = int(row[0])
         articleId = int(row[1])
         articleUrl = ""
-        author = self.__authorsReader.lookupAuthorname(row[2])
```

Comment:
where do you implement this method?

Reply:
in [line 91](https://github.com/CodeLionX/CommentSearchEngine/pull/64/files/79a7a231b80ef6f18179dca06a84b7cce412e927#diff-02e8d794997df615156fa42cb8ba1267R91) and [line 92](https://github.com/CodeLionX/CommentSearchEngine/pull/64/files/79a7a231b80ef6f18179dca06a84b7cce412e927#diff-02e8d794997df615156fa42cb8ba1267R92)